cornice_swagger/swagger.py

```diff
@@ -33,11 +35,18 @@ def __init__(self, services, def_ref_depth=0, param_ref=False, resp_ref=False):
             Defines if swagger responses should be put inline on the operation
             or on the responses section and referenced by JSON pointers.
             Default is inline.
+        :param custom_type_converters:
+            A dictionary mapping user defined cornice types to callables
+            implementing cornice_swagger.converters.schema.TypeConverter iface
+        :param default_type_converter:
+            Default TypeConverter to use when there is no type registered for a
+            cornice type. If it's not given and the type is not registered,
+            cornice_swagger.converters.exceptions.NoSuchConverter is raised
         """
 
         self.services = services
-
-        self.definitions = DefinitionHandler(ref=def_ref_depth)
+        typ_dispatcher = TypeConversionDispatcher(custom_type_converters, default_type_converter)
```

Comment:
My guess is that we should do the same for the parameter dispatcher here, even if we don't take any additional parameters related to it on the `__init__` call. Just to keep semantics.